Client/Frontend/Components/ActionFooterView.swift

```diff
@@ -0,0 +1,74 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import UIKit
+
+final class ActionFooterView: UIView, ThemeApplicable {
+    private struct UX {
+        static let labelSize: CGFloat = 13
+        static let buttonSize: CGFloat = 13
+        static let padding: CGFloat = 8
+        static let containerMargins = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
+    }
+
+    private lazy var titleLabel: UILabel = .build { label in
+        label.font = DefaultDynamicFontHelper.preferredFont(
+            withTextStyle: .footnote,
+            size: UX.labelSize)
+        label.numberOfLines = 0
+        label.setContentCompressionResistancePriority(.required, for: .vertical)
+        label.accessibilityIdentifier = AccessibilityIdentifiers.ActionFooter.title
+    }
+
+    private lazy var primaryButton: ResizableButton = .build { button in
+        button.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(
+            withTextStyle: .footnote,
+            size: UX.buttonSize)
+        button.titleLabel?.numberOfLines = 1
+        button.buttonEdgeSpacing = 0
+        button.setContentCompressionResistancePriority(.required, for: .vertical)
+        button.accessibilityIdentifier = AccessibilityIdentifiers.ActionFooter.primaryAction
+    }
+
+    private lazy var stackView: UIStackView = .build { stackView in
+        stackView.addArrangedSubview(self.titleLabel)
```

Comment:
Sure. I don't like this .build method anyway, it's not more useful that creating the lazy vars. Plus on lazy vars we can be sure the call is marked as noescape, here it might probably create a retain cycle. But if `self` is not used it does not mean it's not implicitly used.

Reply:
The `.build` method was found useful to avoid having to set `translatesAutoresizingMaskIntoConstraints = false` on all UI elements that we create. If you have a better method, let us know - In the meantime, I would avoid having `self.` usage in that lazy var. I personnaly also have found it benefitial to have a `setupLayout` method where we can see how views are added and constrainted at a glance

Reply:
Useful but misleading by hiding implementation details. There's no mention of actually setting `translatesAutoresizingMaskIntoConstraints` in the name so you have to learn and guess it does it. Since we agreed to make it easier for the new developers that contribute to the open source, my suggestion is to go with the standard and simple way of using lazy vars.

Reply:
Not to mention the PR police that will tell you every time to change to build because "this is how we do it" when if fact, not using it is still fine.

Reply:
While it's great to hear that our team values open collaboration and finding solutions together, I believe it's essential to address the specific topic at hand as well by mentioning my opinion. To ensure we're aligned, I will bring this discussion during the engineering sync.

Reply:
PR updated following projects practices.